frontend/src/services/server.ts

```diff
@@ -0,0 +1,25 @@
+import Server from "../models/server/server";
+import axios from "axios";
+
+class Servers {
+  // Function to retrieve all servers + companies
+  async getAll(): Promise<Server[]> {
+    const response = await axios.get<Server[]>(
+      `${import.meta.env.VITE_REST_SERVER_URL}/api/servers`
```

Comment:
should be ` `${import.meta.env.VITE_REST_SERVER_URL}/api/servers/servers` ` since this is how you route the backend